index.html

```diff
@@ -0,0 +1,114 @@
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+        h1 {
+            color: #333;
+        }
+        .search-container {
+            margin-bottom: 20px;
+        }
+        .search-container input {
+            padding: 10px;
+            margin-right: 10px;
+            border: 1px solid #ddd;
+            border-radius: 5px;
+            font-size: 16px;
+        }
+        .search-container button {
+            padding: 10px 20px;
+            border: none;
+            background-color: #007bff;
+            color: white;
+            border-radius: 5px;
+            cursor: pointer;
+            font-size: 16px;
+        }
+        .search-container button:hover {
+            background-color: #0056b3;
+        }
+        .grid {
+            display: grid;
+            grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+            gap: 20px;
+            width: 100%;
+            max-width: 1200px;
+        }
+        .grid-item {
+            background-color: white;
+            padding: 20px;
+            border: 1px solid #ddd;
+            border-radius: 5px;
+            box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+            text-align: center;
+        }
+        .grid-item strong {
+            display: block;
+            margin-bottom: 10px;
+            font-size: 18px;
+        }
+        .grid-item span {
+            display: block;
+            font-size: 16px;
+            color: #555;
+        }
+    </style>
+</head>
+<body>
+    <h1>Currency Rates</h1>
+    <div class="search-container">
+        <input type="text" id="searchFrom" placeholder="From Currency">
+        <input type="text" id="searchTo" placeholder="To Currency">
+        <button onclick="searchCurrencyRate()">Search</button>
+    </div>
+    <div id="currencyGrid" class="grid"></div>
+
+    <script>
+        const currencyRates = [
+            { from: 'USD', to: 'EUR', rate: 0.85 },
+            { from: 'EUR', to: 'USD', rate: 1.18 },
+            { from: 'USD', to: 'GBP', rate: 0.75 },
+            { from: 'GBP', to: 'USD', rate: 1.33 },
+            { from: 'USD', to: 'JPY', rate: 110.53 },
+            { from: 'JPY', to: 'USD', rate: 0.009 },
+            // Add more as needed
+        ];
+
+        function displayCurrencyRates(rates) {
+            const grid = document.getElementById('currencyGrid');
+            grid.innerHTML = '';
+            rates.forEach(rate => {
+                const item = document.createElement('div');
+                item.className = 'grid-item';
+                item.innerHTML = `
+                    <strong>${rate.from} to ${rate.to}</strong>
+                    <span>Rate: ${rate.rate}</span>
+                `;
+                grid.appendChild(item);
+            });
+        }
+
+        function searchCurrencyRate() {
+            const from = document.getElementById('searchFrom').value.toUpperCase();
```

Comment:
```suggestion
            const from = document.getElementById('searchFrom').value.toUpperCase().trim();
```
As mentioned [in the last PR](https://github.com/Merongk/js-projects-currency-converter/pull/2#discussion_r1675568941).